vignettes/MetaboCoreUtils.Rmd

```diff
@@ -546,6 +546,100 @@ Generally, injecting study samples in random order can reduce (or even avoid)
 influence of any related technical bias in the downstream analysis and is highly
 suggested to improve and assure data quality.
 
+## Filtering data: Identifying measurement error 
+
+When dealing with metabolomics results, it is often necessary to filter
+features based on certain criteria. These criteria are typically derived
+from statistical formulas applied to full rows of data, where each row
+represents a feature. In this tutorial, we'll explore a set of functions
+designed for filtering metabolomics data.
```

Comment:
maybe change to something similar to:

...designed to calculate basic quality assessment metrics on which metabolomics data can subsequently be filtered.